easybuild/tools/modules.py

```diff
@@ -555,13 +556,35 @@ def mod_exists_via_show(mod_name):
             # - Check first non-whitespace line for something that looks like an absolute path terminated by a colon
             mod_exists_regex = r'\s*/.+:\s*'
             for line in stderr.split('\n'):
+                # skip whitespace lines
+                self.log.debug("Checking line '%s' to determine whether %s exists...", line, mod_name)
                 if OUTPUT_MATCHES['whitespace'].search(line):
+                    self.log.debug("Treating line '%s' as whitespace, so skipping it", line)
                     continue
+
+                # if any errors occured, conclude that module doesn't exist
                 if OUTPUT_MATCHES['error'].search(line):
+                    self.log.debug("Line '%s' looks like an error, so concluding that %s doesn't exist", mod_name)
```

Comment:
Missing format argument

Reply:
fixed missing format argument (but kept the comment, still helps with explaining the logic)